### PR TITLE
Update Jakarta Data 1.0.0-b3 

### PR DESCRIPTION
**Pull request by otaviojava**
# Context

Update Jakarta Data 1.0.0-b3 that enhances documentation.

Include the following in the PR:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [x] Specification PDF in the form of wombat-spec-x.y.pdf
- [x] Specification HTML in the form of wombat-spec-x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md

### CODE REVIEW

**Comment by otaviojava** [comment]
@ivargrimstad, thank you Ivar, I removed it.

**Review by ivargrimstad — APPROVED** [review]

File: data/1.0/_index.md [hunk 1]
```diff
@@ -44,11 +44,11 @@ garage.save(ferrari);
 
 
 * [Jakarta Data Release Record](https://projects.eclipse.org/projects/ee4j.data/releases/1.0)
-* [Jakarta NoSQL 1.0 Specification Document](./data-1.0.0-b2.pdf) (PDF)
-* [Jakarta NoSQL 1.0 Specification Document](./data-1.0.0-b2.html) (HTML)
+* [Jakarta NoSQL 1.0 Specification Document](./data-1.0.0-b3.pdf) (PDF)
+* [Jakarta NoSQL 1.0 Specification Document](./data-1.0.0-b3.html) (HTML)
```
Comment (lukasj) [inline data/1.0/_index.md:48]: @otaviojava just came across this one on the live website - see that the link says *NoSQL* while it points to and actually is *Data*

**Comment by ivargrimstad** [comment]
I'll fix it directly. It's just the text for the links that are incorrect

File: data/1.0/_index.md (hunk 1 above)
Comment (otaviojava) [reply data/1.0/_index.md:48]: ops, my bad.
Thank you @ivargrimstad

**Pull request merged**